baseline-module-jvm-args-compiler-plugins/src/main/java/com/palantir/baseline/modulejvmargs/compilerplugins/AllowReleaseAndAddExportsToBeUsedTogetherByChangingCompilerInternalsUsingReflection.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.modulejvmargs.compilerplugins;
+
+import com.google.auto.service.AutoService;
+import com.sun.source.util.JavacTask;
+import com.sun.source.util.Plugin;
+import com.sun.tools.javac.api.BasicJavacTask;
+import com.sun.tools.javac.comp.Modules;
+import com.sun.tools.javac.util.Context;
+import java.lang.reflect.Field;
+
+/**
+ * The JDK devs arbitrarily decided that you cannot use the `--release` arg along with `--add-exports`
```

Comment:
nit: 
```suggestion
 * The JDK devs arbitrarily decided that you cannot use the {@code --release} arg along with {@code --add-exports}
```

Unfortunately we're not on java 23 :'(